lineage/tests/test_import_lineage.py

```diff
@@ -62,3 +64,12 @@ def test_MCF10A(self):
         np.testing.assert_allclose(lin1[0].obs, [1, 30.0, 1], rtol=1e-2)
         np.testing.assert_allclose(lin1[1].obs, [np.nan, 17.5, 1], rtol=1e-2)
         np.testing.assert_allclose(lin1[2].obs, [np.nan, 17.5, 1], rtol=1e-2)
+
+    def test_bic(self):
```

Comment:
Do you know why the test is failing?

Reply:
oh.. I have added the distance and velocity to the observations and I forgot to apply those changes in the test. I fixed it.